### DISCUSSION TITLE
Dark Mode Theme

### DISCUSSION THREAD

**Discussion by VaibhavSharma24** [avelino/awesome-go#5084] [Ideas]
I want to add dark mode toggling button to your site to make it more readable and build a great user experience

**Comment by avelino** [comment]
I like the idea, we need to improve our UX/UI layout a lot, supporting a dark theme would be great

@VaibhavSharma24 can you contribute to this improvement?

**Reply by VaibhavSharma24** [reply]
@avelino I will be more than happy to work on this feature and add my contribution to your amazing project 😊
